src/snapshot/tosvg.js

```diff
@@ -107,7 +107,7 @@ module.exports = function toSVG(gd, format) {
 
     // remove draglayer for Adobe Illustrator compatibility
     if(fullLayout._draggers) {
-        fullLayout._draggers.node().remove();
+        fullLayout._draggers.remove();
```

Comment:
`ChildNode.remove` is a thing though: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

but not in IE.